.travis.yml

```diff
@@ -6,7 +6,7 @@
   go:
     - 1.6.2
   install: make deps
-  script: make .gitvalidation && make validate && make test
+  script: make .gitvalidation && make validate && make test && make test-skopeo
```

Comment:
I'm not aware on anything on Travis (can be done with plain Jenkins though). Might want to read something on the Travis doc but I agree it isn't critical